### PR TITLE
Change player's facing when they click on something

### PR DESCRIPTION
**Pull request by SpaceManiac**

### CODE REVIEW

File: Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs [hunk 1]
```diff
@@ -409,6 +410,8 @@ private void UserInteraction(IEntity player, GridCoordinates coordinates, Entity
                 return;
             }
 
+            playerTransform.LocalRotation = new Angle(coordinates.ToMapPos(_mapManager) - playerTransform.MapPosition.Position);
```
Comment (PJB3005) [inline Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs]: Probably best to directly round this to the nearest cardinal direction for now?

While right now the sprite system will automatically round it, that won't forever be the case, and there might be other funky things happening due to this in the future (bounding boxes rotating).

Comment (SpaceManiac) [reply Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs]: Movement code sets this 8-directional, do you want that or 4-directional rounding

Comment (PJB3005) [reply Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs]: Oh, hrm. in that case it's fine I guess.

Comment (SpaceManiac) [reply Content.Server/GameObjects/EntitySystems/Click/InteractionSystem.cs]: I agree that it would make sense to return to later

**Pull request merged**